PowerEditor/src/WinControls/ProjectPanel/TreeView.cpp

```diff
@@ -118,13 +118,20 @@ void TreeView::removeItem(HTREEITEM hTreeItem)
 	// Deallocate all the sub-entries recursively
 	cleanSubEntries(hTreeItem);
 
+
+
 	// Deallocate current entry
 	TVITEM tvItem;
 	tvItem.hItem = hTreeItem;
 	tvItem.mask = TVIF_PARAM;
 	SendMessage(_hSelf, TVM_GETITEM, 0,(LPARAM)&tvItem);

```

Comment:
You ***absolutely*** should check to see if `::SendMessage` returned `FALSE` before using `tvItem.lParam`.